### DISCUSSION TITLE
How can I put data into a binary type column?

### DISCUSSION THREAD

**Discussion by lsw8724** [questdb/questdb#1173] [Ideas]
Hi,

I'm trying to manually put data into a binary type column using web console, InfluxDB line protocol and REST.
But I couldn't find any reference on this method. It seems that it can't simply be inserted as a hex string.
Is there any way to put data into a binary column?

Thanks.

**Comment by bsmth** [comment]
Thanks for opening the discussion @lsw8724 , there is an issue to track the implementation here: https://github.com/questdb/questdb/issues/1187
